tclaim.go

```diff
@@ -228,8 +228,8 @@ var (
 	}
 )
 
-// TrustTier provides the trust tier bucket of the trustworthiness claim
-func (o TClaim) TrustTier(color bool) string {
+// trustTier provides the trust tier bucket of the trustworthiness claim
+func (o TClaim) trustTier(color bool) string {
```

Comment:
Why has this been hidden? It seems that being able to easily get trust tier from a claim (and the associated `Is` checks below) would be generally useful?

Reply:
My plan was to only expose this through the Report() method.  But yeah, it'd be a tad myopic to focus on just use case (i.e., that of a CLI parsing the AR).  Will change, thanks for the input.

Reply:
done in 10718e5